OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -69,7 +69,7 @@ class AttackActivity : Activity
 
 			public AttackActivity(Actor self, Target target, bool allowMove, bool forceAttack)
 			{
-				attack = self.Trait<AttackFollow>();
+				attack = self.TraitsImplementing<AttackFollow>().Where(af => !af.IsTraitDisabled).First();
```

Comment:
`attack = self.TraitsImplementing<AttackFollow>().First(af => !af.IsTraitDisabled)`

Also, this will crash if they're all disabled. Intentional?